CMakeLists.txt

```diff
@@ -124,8 +124,12 @@ endif()
 
 set(OPTIMIZATION_FLAGS "")
 
-set(CMAKE_C_FLAGS "-g ${COMMON_FLAGS} ${OPTIMIZATION_FLAGS} -DNDEBUG -DDEBUGVM=0")
-set(CMAKE_CXX_FLAGS "-g ${COMMON_FLAGS} ${OPTIMIZATION_FLAGS} -DNDEBUG -DDEBUGVM=0")
+set(CMAKE_C_FLAGS "-g ${COMMON_FLAGS} ${OPTIMIZATION_FLAGS}")
+set(CMAKE_CXX_FLAGS "-g ${COMMON_FLAGS} ${OPTIMIZATION_FLAGS}")
+
+if(NOT ${CONFIG} MATCHES "Debug")
+  set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -DNDEBUG -DDEBUGVM=0")
+endif()
```

Comment:
This particular change is breaking the build. It seems the Cog VM is not working without those flags. @tesonep any idea?